tethysext/atcore/tests/integrated_tests/models/files_database/file_collection_client_tests.py

```diff
@@ -463,3 +474,327 @@ def test_bad_collection_client(self):
             _ = database_client.instance
         self.assertTrue(f'FileCollection with id "{str(collection_id)}" not found.' in str(exc.exception))
+
+    def test_export_item_new_name(self):
+        """Test exporting an item with a new name."""
+        test_dir_name = 'test_export_item_new_name'
+        base_files_root_dir = os.path.join(self.test_files_base, test_dir_name)
+        root_dir = os.path.join(self.test_files_base, 'temp', test_dir_name)
+        self.copy_files_to_temp_directory(base_files_root_dir, root_dir)
+        _, collection_instance = self.get_database_and_collection(
+            database_id=self.general_database_id, collection_id=self.general_collection_id,
+            root_directory=root_dir, database_meta={}, collection_meta={}
+        )
+        collection_client = FileCollectionClient(self.session, self.general_collection_id)
+        exported_file = os.path.join(root_dir, 'exported', 'exported_file.txt')
+        collection_client.export_item('file1.txt', exported_file)
+        self.assertTrue(os.path.exists(exported_file))
+
+    def test_export_item_does_not_exist(self):
+        """Test exporting an item that does not exist throws the correct error."""
+        test_dir_name = 'test_export_item_does_not_exist'
+        base_files_root_dir = os.path.join(self.test_files_base, test_dir_name)
+        root_dir = os.path.join(self.test_files_base, 'temp', test_dir_name)
+        self.copy_files_to_temp_directory(base_files_root_dir, root_dir)
+        _, collection_instance = self.get_database_and_collection(
+            database_id=self.general_database_id, collection_id=self.general_collection_id,
+            root_directory=root_dir, database_meta={}, collection_meta={}
+        )
+        collection_client = FileCollectionClient(self.session, self.general_collection_id)
+        exported_file = os.path.join(root_dir, 'exported', 'file1.txt')
+        with self.assertRaises(FileCollectionItemNotFoundError) as exc:
+            collection_client.export_item('file1.txt', exported_file)
+        self.assertTrue('"file1.txt" not found in this collection.' in str(exc.exception))
+
+    def test_export_item_to_directory(self):
+        """Test exporting a file to a non existent directory."""
+        test_dir_name = 'test_export_item_new_name'
+        base_files_root_dir = os.path.join(self.test_files_base, test_dir_name)
+        root_dir = os.path.join(self.test_files_base, 'temp', test_dir_name)
+        self.copy_files_to_temp_directory(base_files_root_dir, root_dir)
+        _, collection_instance = self.get_database_and_collection(
+            database_id=self.general_database_id, collection_id=self.general_collection_id,
+            root_directory=root_dir, database_meta={}, collection_meta={}
+        )
+        collection_client = FileCollectionClient(self.session, self.general_collection_id)
+        export_dir = os.path.join(root_dir, 'exported')
```

Comment:
Seems like there is a pretty good chance that 'exported' directory exists, no?

Reply:
NVM. copy_files_to_temp_directory ends up removing it, no? May want to implement `tearDown` so this is explicit.

Reply:
I thought about that. But i kind of wanted it to stick down between runs for verification purposes. Also not all of the tests use this temp area method. So it would be overhead for ever test, even though they don't all use it. If you want it I will do it though.